lapack.go

```diff
@@ -108,6 +108,17 @@ const (
 	SVDNone      SVDJob = 'N' // Do not compute singular vectors
 )
 
+// GSVDJob specifies the singular vector computation type for GSVD.
+type GSVDJob byte
+
+const (
+	GSVDU    GSVDJob = 'U' // Compute orthogonal matrix U
+	GSVDV    GSVDJob = 'V' // Compute orthogonal matrix V
+	GSVDQ    GSVDJob = 'Q' // Compute orthogonal matrix Q
+	GSVDUnit GSVDJob = 'I' // Use unit-initialized matrix
+	GSVDNone GSVDJob = 'N' // Do not compute orthogonal matrix
```

Comment:
I don't insist on `lapack.None` either but we should unify. Now eigenvalues use `lapack.None`, SVD has `lapack.SVDNone`, here we have `lapack.GSVDNone`. There is #163 where this discussion could belong.